var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -37,7 +37,10 @@ class QuantumEspresso(Package):
 
     # Support for HDF5 has been added starting in version 6.1.0 and is
     # still experimental, therefore we default to False for the variant
-    variant('hdf5', default=False, description='Builds with HDF5 support')
+    variant(
```

Comment:
I don't quite remember if none was a magic variant value or not. @alalazo I seem to remember you implemented something there?

Reply:
I was following the example here:
https://spack.readthedocs.io/en/latest/packaging_guide.html#complex-validation-logic-for-variant-values

You are allow to use '`none'` but not Python's `None`.